demo/build.gradle

```diff
@@ -19,7 +19,7 @@ android {
         versionCode 1
         versionName "1.0"
 
-        manifestPlaceholders = [downloadAuthority: "${applicationId}"]
+        manifestPlaceholders = [downloadAuthority: "${applicationId}.downloads"]
```

Comment:
how the authority is determined can be more sophisticated. I think convention is still the best way to go.